bfabric/src/result_container.py

```diff
@@ -1,6 +1,8 @@
-
 from enum import Enum
 
+from bfabric.src.suds_format import suds_asdict_recursive
+
+
 class BFABRIC_RESULT_TYPE(Enum):
```

Comment:
It would be more Pythonic to name enum classes in SnakeCase as well, variants with caps is fine since they are "constants":
(This would require change in several places, I'd just rename it in an IDE)

```suggestion
class BfabricResultType(Enum):
```

Reply:
I will never unlearn C, but I promise to try :D